include/zephyr/xen/dom0/uimage.h

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
drop the file from zephyr and move this file to xen-lib 
@firscity @oleksiimoisieiev